### PR TITLE
If -s flag is specified, that mean we want it to override config file.

### PR DESCRIPTION
**Pull request by TinLe**
## Pull Request Checklist

**Is this in reference to an existing issue?**
No

#### General

- [ ] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### Purpose
Fix an issue preventing when specifically specifying the socket location on cli it looks in your ini config.
#### Known Compatibility Issues
none

### CODE REVIEW

**Comment by majormoses** [comment]
Thanks for your contribution to Sensu plugins! Without people like you submitting PRs we couldn't run the project. I will review it shortly.

File: bin/check-mysql-status.rb [hunk 1]
```diff
@@ -115,7 +115,11 @@ def credentials
       section = ini[config[:ini_section]]
       db_user = section['user']
       db_pass = section['password']
-      db_socket = section['socket']
+      if config[:socket]
```
Comment (majormoses) [inline bin/check-mysql-status.rb]: Rubocop is telling us that it is better to use the return of the conditional for conditional assignment this looks like this:
```ruby
db_socket = if config[:socket]
              config[:socket]
            else
              section['socket']
            end
```

**Comment by TinLe** [comment]
Thank you for the review.  I have updated the change as suggested and added CHANGELOG entry.

**Comment by majormoses** [comment]
Looks like you got three spaces for tabs rather than two I will fix this up.

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-mysql/versions/2.2.1

**Pull request merged**